File: back-end/src/main/java/backend/model/po/Course.java
package backend.model.po;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import lombok.Data;

/**
 *
 * @TableName course
 */
@TableName(value ="course")
@Data
public class Course implements Serializable {
    /**
     *
     */
    @TableId(type = IdType.AUTO)
    private Integer courseId;

    /**
     *
     */
    private String name;

    /**
     *
     */
    private String serialNum;

    /**
     *
     */
    private Integer stuTime;

    /**
     *
     */
    private Integer score;

    /**
     *
     */
    private String semester;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}

File: front-end/src/mock/course.js
import Mock from "mockjs";

const myCourses = [
    {
        'courseId': 1,
        'name': '操作系统',
        'serialNum':'01',
        'clazzNum':2,
        'score':4,
        'stuTime':24,
        'semester':'2-up',
    },
    {
        'courseId': 2,
        'name': '数据库原理',
        'serialNum':'02',
        'clazzNum':3,
        'score':6,
        'stuTime':36,
        'semester':'2-down',
    },
]
Mock.mock(`/course/list`, 'get', (options) => {
    return{
        code:200,
        data:myCourses
    }
})
